src/main/app/form/validation.ts

```diff
@@ -121,7 +121,7 @@ export const isEmailValid: Validator = value => {
 };
 
 export const isFieldLetters: Validator = value => {
-  if (!(value as string).match(/^[a-zA-Z][a-zA-Z\s]*$/)) {
+  if (!(value as string).match(/^[\p{Script=Latin}'’\-\s]*$/gu)) {
```

Comment:
interesting, what about welsh, are all there letters considered Latin letters?

Reply:
[Welsh uses Latin letters](https://en.wikipedia.org/wiki/Welsh_orthography) so we should be okay `\p{Script=Latin}` includes what was here previously `a-zA-Z` but also includes [letters with accents](https://en.wikipedia.org/wiki/Diacritic) e.g. `éâßáêè` etc.